templates/todo/tag_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <h1>Delete tag</h1>
+
+
+  <p>Are you sure you want to delete the tag {{ tag }}</p>
```

Comment:
Add a question mark on question:

```suggestion
  <p>Are you sure you want to delete the tag {{ tag }}?</p>
```